<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>516</width>
           <height>475</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="ImageLabel" name="imageLabel">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="coordinatesLabel">
        <property name="text">
         <string>&amp;Coordinates</string>
        </property>
        <property name="buddy">
         <cstring>coordinatesLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="coordinatesLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="analysisMenu">
    <property name="title">
     <string>&amp;Analysis</string>
    </property>
    <addaction name="pathLengthAction"/>
    <addaction name="polygonAreaAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="analysisMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="separator"/>
   <addaction name="zoomInAction"/>
   <addaction name="zoomOutAction"/>
   <addaction name="separator"/>
   <addaction name="pathLengthAction"/>
   <addaction name="polygonAreaAction"/>
   <addaction name="separator"/>
   <addaction name="clearAction"/>
  </widget>
  <action name="openAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open.svg</normaloff>:/images/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/about.svg</normaloff>:/images/about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/qt.svg</normaloff>:/images/qt.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom-in.svg</normaloff>:/images/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom-out.svg</normaloff>:/images/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="pathLengthAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/path.svg</normaloff>:/images/path.svg</iconset>
   </property>
   <property name="text">
    <string>Path &amp;Length</string>
   </property>
   <property name="toolTip">
    <string>Path Length</string>
   </property>
  </action>
  <action name="polygonAreaAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/area.svg</normaloff>:/images/area.svg</iconset>
   </property>
   <property name="text">
    <string>Polygon &amp;Area</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/clear.svg</normaloff>:/images/clear.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/settings.svg</normaloff>:/images/settings.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>ImageLabel</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
